.gitignore

```diff
@@ -66,3 +66,4 @@ target/
 /.vagrant
 *.sqlite3
 bower_components
+.DS_Store
```

Comment:
If you put `. DS_Store` in your `~/.gitignore_global`, you won't need to add it to the project's `.gitignore`.  It's better to keep the entries in this file specific to the project/build and not add OS-specific items.